pytorch_lightning/core/optimizer.py

```diff
@@ -162,3 +164,216 @@ def closure_dis():
         assert trainer is not None
         with trainer.profiler.profile(profiler_action):
             trainer.training_type_plugin.optimizer_step(self._optimizer, self._optimizer_idx, closure, **kwargs)
+
+
+def _init_optimizers_and_lr_schedulers(model: "pl.LightningModule") -> Tuple[List, List, List]:
+    prev_fx_name = model._current_fx_name
+    model._current_fx_name = "configure_optimizers"
+    model.trainer._lightning_optimizers = None
+    optim_conf = model.configure_optimizers()
+    model._current_fx_name = prev_fx_name
```

Comment:
this part looks like it was changed

Reply:
Yeah, I can't remember why I thought that was necessary, but putting it back now to how it was before